.github/workflows/on_release.yml

```diff
@@ -4,7 +4,7 @@ on:
     # job will automatically run after a new "release" is create on github.
     types: [created]
 
```

Comment:
If we don't want the release to be automatically created, can we at least make this workflow triggered when release is created? (rather than manualy?)

Reply:
Yes, we can